docs/concepts/ipfs-implementations.md

```diff
@@ -5,6 +5,13 @@ description: "There isn't just one canonical IPFS implementation. Learn about th
 
 # IPFS Implementations
 
+:::warning js-ipfs being discontinued
+Development of the [js-ipfs project](https://github.com/ipfs/js-ipfs) is being discontinued to focus on [Helia](https://github.com/ipfs/helia), a leaner, more modular, modern implementation of IPFS in JavaScript slated for release in 2023. To learn more about Helia and the current state of IPFS in JS, see the [blog post](https://blog.ipfs.tech/state-of-ipfs-in-js/). 
```

Comment:
I would move this either to below the table or into the js-ipfs row so it's not the first thing you read on the page.

Also, this is the first time I've seen `slated` used this way. Looking it up reveals that it's a north American word which probably explains why I haven't seen it. What do you think about a more neutral verb, e.g. scheduled or planned?